src/components/ForgetPassword.tsx

```
@@ -0,0 +1,110 @@
+              <div className='flex'>
+                <div className='flex justify-center'>
+                  <CheckCircle2 className='h-12 w-12 text-green-500' />
+                </div>
+                <p className='text-center text-base text-gray-600'>
+                  We've sent an email to {email.replace(/(.{2})(.*)(?=@)/, (_, a, b) => a + '*'.repeat(b.length))} with
+                  password reset instructions. Please check your email.
+                </p>
+              </div>
+
+              <Button
+                onClick={() => (window.location.href = '/login')}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)